.github/container/Dockerfile.jax

```diff
@@ -36,7 +36,7 @@ RUN --mount=type=ssh \
     --mount=type=secret,id=SSH_KNOWN_HOSTS,target=/root/.ssh/known_hosts \
 <<"EOF" bash -ex
     git-clone.sh ${URLREF_JAX} ${SRC_PATH_JAX}
-    sed 's/^numpy.*/numpy<2.0.0/' ${SRC_PATH_JAX}/build/requirements.in
+    sed -i 's/^numpy.*/numpy<2.0.0/' ${SRC_PATH_JAX}/build/requirements.in
```

Comment:
Can we add a comment above this to document the reason and also prompt future actions to remove it once TF supports NumPy 2?

Reply:
Looks like we don't it any more. In TF requirements for python3.12 there is `numpy==2.1.1`